src/Dialogs/UnsafePasteDialog.vala

```diff
@@ -70,6 +70,8 @@ public class PantheonTerminal.UnsafePasteDialog : Gtk.Dialog {
         add_action_widget (cancel_button, 1);
         add_action_widget (ignore_button, 0);
 
+        show_protection_warnings.set_active (true);
```

Comment:
If this i bound to the setting, does it need to be explicitly marked as active?

Reply:
I tested locally and it appears to still properly check itself when bound to the setting, so I believe this should be removed.

Reply:
I added it so the dialog still has the same default behaviour: Showing the dialog if the CheckButton is untouched by the user. The old phrasing requires the user to act to mute the dialog. The new phrasing should still require the user to act.

I think this is required to fulfill the requirement "and have it checked by default." that your made in the issues description.

Sorry in advance if maybe I don't get what your mean.

Reply:
@ls42 No worries, it's kind of confusing! But by using a binding to the setting, the check will automatically check itself if that underlying setting is active. The setting is already active by default (controlled by the default GSchema), and it must be active to show the dialog in the first place, so this line is just unnecessary.

Reply:
Alright! Very interesting. I remove the line and it's still working.